modules/core/src/lib/effect-manager.js

```diff
@@ -25,25 +27,28 @@ export default class EffectManager {
     return redraw;
   }
 
-  setEffects(effects = []) {
-    this.effects = effects;
-  }
-
   getEffects() {
-    return this.effects;
+    let effects = this.effects;
+    if (this.needApplyDefaultLighting) {
+      effects = this.effects.slice();
+      effects.push(this.defaultLightingEffect);
+    }
+    return effects;
   }
 
   // Private
-  applyDefaultLightingEffect() {
+  setEffects(effects = []) {
```

Comment:
our code base uses `_` prefix indicates private methods?

Reply:
not necessary from what I know, we just need add Private comment before the private functions